packages/web-components/fast-foundation/src/switch/switch.template.ts

```diff
@@ -26,7 +27,12 @@ export function switchTemplate<T extends FASTSwitch>(
                         ? "label"
                         : "label label__hidden"}"
             >
-                <slot ${slotted("defaultSlottedNodes")}></slot>
+                <slot
+                    ${slotted({
+                        property: "defaultSlottedNodes",
+                        filter: whitespaceFilter,
```

Comment:
I might have missed it - did we add the whitespace filter to every instance where a label gets slotted? Is there a value add of including the filter here besides for the sake of our tests or how the suite manages comparing strings?